app/kumactl/cmd/install/testdata/install-control-plane.local.golden.yaml

```diff
@@ -4583,6 +4584,20 @@ spec:
   selector:
     app: kuma-control-plane
 ---
+apiVersion: v1
+kind: Service
+metadata:
+  name: global-local-sync
+  namespace: kuma-system
+spec:
+  type: NodePort
+  ports:
+    - port: 5681
+      nodePort: 30681
+      name: global-local-sync
+  selector:
+    app: kuma-control-plane
+---
```

Comment:
Is it needed? We should not expose API Server globally. If this is temporary for a time that we don't have sync, let's make sure we remove this after introducing sync.

Reply:
Yes, this is a dedicated service that exposes API for the purposes of Local registration to Global (actually the verification that Local is there). But it should be replaced with the real Sync Port/Service that @lobkovilya is bringing in.

Reply:
Let's create a task of something so we don't forget to remove it.